backend/internal/common/env_config.go

```diff
@@ -166,23 +126,9 @@ func parseEnvConfig() error {
 	case "":
 		EnvConfig.KeysStorage = "file"
 	case "database":
-		// Resolve encryption key using the same pattern
-		encryptionKey, err := resolveStringOrFile(
-			EnvConfig.EncryptionKey,
-			EnvConfig.EncryptionKeyFile,
-			"ENCRYPTION_KEY",
-			// Do not trim spaces because the file should be interpreted as binary
-			false,
-		)
-		if err != nil {
-			return err
+		if EnvConfig.EncryptionKey == "" {
```

Comment:
I just noticed this in the PR that was merged. For the encryption key, when we read it from file, we need to treat it as binary, so we shouldn't be putting it into a `string`.

Env vars can be treated as strings, but when we read from file it should be treated as `[]byte` - this is because it's possible people pass binary files (like with `head -c 32 /dev/random > key_file`).